src/js/brandColors/brandColors.ts

```diff
@@ -2,87 +2,87 @@ import { BrandColors } from './brandColors.types';
 
 export const brandColors: BrandColors = {
   white: {
```

Comment:
Updating casing to uppercase to match with the rest of the javascript color values